### PR TITLE
Fix: Notification center doesn't display playback speed below 1x

### LINKED ISSUE #964: Speed control stops at 2x on media notification
### Description

On the media notification tapping the speed button loops through the speed settings, but stops at 2x. I often use between 2x and 3x and it's frustrating to not be able to control that from the notification, especially if I fat finger it and I have to go back to the app to fix it. 

### Step-by-step reproduction instructions

1. Tap on the speed setting on the media notification
2. Observe that it loops at 2x rather than 3x

### Screenshots or screen recording

_No response_

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Google Pixel 7 Pro with Android 13

### LINKED ISSUE #1745: Notification center doesn't display playback speed below `1x`
### Description

When a podcast is playing with a playback speed below `1x` I expect the notification to correctly show this state.

### Step-by-step reproduction instructions

1. Play a podcast.
2. Open playback effects.
3. Change the playback speed to something below `1x`.
4. Open notification center.
5. Notice that the playback speed shows `1x` instead of the value that you set.

### Screenshots or screen recording

| Settings | Notification |
| - | - |
| Screenshot_20240126-125818 | Screenshot_20240126-125809 |

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Google Pixel 6, Android 14, 06c436a651c5eb04ff6d57481ebdfd5795f6579b

### PR DESCRIPTION
**Pull request by mebarbosa**
## Description
- This PR displays playback speeds below `1x` in notification media.
- Add the ability to update playback speed via playback speed button in notification media from `2x` to `3x`, from `3x` to `0.6x`, from `0.6x` to `0.8x` and finally from `0.8x` to `1.0x`.
- Updates all playback speed resources to make sure they all have the same font family. See: internal reference: `p1707925412314169-slack-pocket-casts-design`.

Closes #1745
Closes #964

###  1. Update playback speed in playback effects
1. Play a podcast.
2. Open playback effects.
3. Change the playback speed to something between.
4. Open notification center.
5. You should see the playback speed updated in notification media center.

###  2. Update playback speed in playback speed notification center button
1. Play a podcast.
2. Open notification center
3. Click on playback speed button (see the video below)
4. You should be able to switch between `0.6x`, `0.8x`, `1.0x`, `1.2x`, `1.4x`, `1.6x`, `1.8x`, `2.0x`, and `3.0x`.

Note: If you are currently at `1.7x` and tap the playback speed notification center button, it will update the speed to `1.8x` since it is the closest option listed above

## Screenshots or Screencast 
[Screen_recording_20240221_091321.webm](https://github.com/Automattic/pocket-casts-android/assets/42220351/d5a0cd77-3e7b-45b0-943d-c1da54fbcedc)

| Automotive |
|--------|
| | 

| App |
|--------|
| | 

## Checklist
- [x] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
 
#### I have tested any UI changes...

- [x] with different themes
- [ ] with a landscape orientation
- [x] with the device set to have a large display and font size
- [ ] for accessibility with TalkBack

### CODE REVIEW

**Review by MiSikora — APPROVED** [review]
I left one minor comment.

I found a race condition I think. Or it is related to the rounding from my comment error but I don't think so. You can see this in the video below. After I tapped to change speed it didn't update from `0.6` to `0.8`. I had to tap it again to make it jump from `0.6` to `1.0`. I can also trigger this bug when changing the speed from the app.


https://github.com/Automattic/pocket-casts-android/assets/30936061/9b56e7f5-a98b-4e79-beed-e06372f59e3b

File: modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/MediaSessionManager.kt [hunk 1]
```diff
@@ -423,30 +423,34 @@ class MediaSessionManager(
...
-                            currentSpeed == 2.4 -> IR.drawable.auto_2_4x
-                            currentSpeed == 2.5 -> IR.drawable.auto_2_5x
-                            currentSpeed == 2.6 -> IR.drawable.auto_2_6x
-                            currentSpeed == 2.7 -> IR.drawable.auto_2_7x
-                            currentSpeed == 2.8 -> IR.drawable.auto_2_8x
-                            currentSpeed == 2.9 -> IR.drawable.auto_2_9x
-                            currentSpeed == 3.0 -> IR.drawable.auto_3x
-                            else -> IR.drawable.auto_1x
+                        val drawableId = when (playbackState.playbackSpeed.roundedSpeed()) {
+                            0.5 -> IR.drawable.auto_0_5
```
[the remark this answers is not in the document]
Comment (mebarbosa) [reply modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/MediaSessionManager.kt]: I can addressed this on this PR for sure. Here: [fa653a1](https://github.com/Automattic/pocket-casts-android/pull/1862/commits/fa653a12a537d2d1a076cc75768288c10fb72a9d)

**Comment by mebarbosa** [comment]
> I left one minor comment.
> 
> I found a race condition I think. Or it is related to the rounding from my comment error but I don't think so. You can see this in the video below. After I tapped to change speed it didn't update from `0.6` to `0.8`. I had to tap it again to make it jump from `0.6` to `1.0`. I can also trigger this bug when changing the speed from the app.
> 
>  foo.mp4

@MiSikora After implementing the range changes I tried to reproduce this, but I could not see this scenario. Could you test it again?

**Comment by MiSikora** [comment]
Reproduction rate is very small but I'll try again tomorrow.

**Comment by CookieyedCodes** [comment]
I am also going to suggest you guys look to see if this is linked to carplay just because the notification center shares the order of what appears in carplay and their probably should be less taps on carplay due it to being not wanting to be a distraction on cars 🤔

**Comment by MiSikora** [comment]
@CookieyedCodes I think is tracked by #938

**Review by MiSikora — COMMENTED** [review]
Sorry wrong PR. 😅

**Comment by CookieyedCodes** [comment]
> @CookieyedCodes I think it is tracked in #938

Gud gud I was just remembering the weird quirks why android 13 media controls were weirdly tied to carplay, 

I actually have designed a sider I will get David to eventually give more feedback on 
https://github.com/Automattic/pocket-casts-android/discussions/1452 so I have considered #938

**Pull request merged**